pkg/limitador/k8s_objects.go

```diff
@@ -1,21 +1,33 @@
 package limitador
 
 import (
+	"crypto/md5"
+	"fmt"
 	limitadorv1alpha1 "github.com/kuadrant/limitador-operator/api/v1alpha1"
 	"github.com/kuadrant/limitador-operator/pkg/helpers"
 	appsv1 "k8s.io/api/apps/v1"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	"sigs.k8s.io/yaml"
 )
 
 const (
-	DefaultVersion         = "latest"
-	DefaultReplicas        = 1
-	Image                  = "quay.io/3scale/limitador"
-	StatusEndpoint         = "/status"
-	DefaultServiceHTTPPort = 8080
-	DefaultServiceGRPCPort = 8081
+	DefaultVersion             = "latest"
+	DefaultReplicas            = 1
+	Image                      = "quay.io/3scale/limitador"
+	StatusEndpoint             = "/status"
+	DefaultServiceHTTPPort     = 8080
+	DefaultServiceGRPCPort     = 8081
+	EnvLimitadorConfigFileName = "LIMITADOR_CONFIG_FILE_NAME"
+	LimitadorCMHash            = "hash"
+	LimitsCMNamePrefix         = "limits-config-"
+	LimitadorCMMountPath       = "/"
+	LimitadorLimitsFileEnv     = "LIMITS_FILE"
+)
+
+var (
+	LimitadorConfigFileName = helpers.FetchEnv(EnvLimitadorConfigFileName, "limitador-config.yaml")
```

Comment:
I do not see the value of having limits file path exposed in some env var. What would be the use case?

Reply:
It's true it doesn't bring an obvious value, it was meant in case there were different files (like list of limits grouped by a specific criteria -- i.e: Basic plan, Premium, Sample, etc...) and you could easily try different ones... anyways, there are other ways to approach the use case. I could remove it